### PR TITLE
deploy to prod, staging using different branch.

### PR DESCRIPTION
**Pull request by Tay-Sike**
Changes:
- deploy to prod
- Staging uses staging branch

### CODE REVIEW

**Comment by Tay-Sike** [comment]
Task linked: [CU-863h8430d [Alerts] Adjust Slack Alerting format and move GQE to plt-prd](https://app.clickup.com/t/863h8430d)

File: infraspec.yaml [hunk 1]
```diff
@@ -6,12 +6,12 @@ deploy:
...
   environments:
   - name: production
-    disabled: true
     releaseStrategy:
+      branch: main
       tagFormat: semver
   - name: staging
     releaseStrategy:
-      branch: main
+      branch: staging
```
[the remark this answers is not in the document]
Comment (Tay-Sike) [reply infraspec.yaml:14]: But I want to remove the webservers from staging and not prod (main branch). Is this how I can do that?

[the remark this answers is not in the document]
Comment (Tay-Sike) [reply infraspec.yaml:14]: Only thing to run in staging would be 1-2 optional test webservers that point to less used gcp projects. `Infrastructure POC` or another project name

[the remark this answers is not in the document]
Comment (vborodaykevych) [reply infraspec.yaml:14]: Then you probably do not need staging, so just deploy prod and disable staging.

**Review by vborodaykevych — APPROVED** [review]

**Pull request merged**